### ISSUE TITLE
Add bibl URIs for references which are not to mss

### ISSUE THREAD

**Issue by wlpotter** [srophe/britishLibrary-data#111]
#44 took care of this for ms references. After #110 has been completed, we will want to add URIs for all of these bibls. This will probably be aided by a spreadsheet and OpenRefine. Not a priority for first release (as it's dependent on #110 which is not a priority).

**Comment by wlpotter** [comment]
In progress from summer 2024 workshop; students helping track down or create Zotero links. Once all Zotero records have been created or found, will need to do the following:

- [ ] find and replace temporary IDs in `bibl/ref/@target` with the CBSS URI
- [ ] find and replace Zotero URLs in `bibl/ref/@target` with the CBSS  URI based on the item key (might be able to do this with a regex, but may need a script)

**Comment by wlpotter** [comment]
This was done except for the few that still need new Zotero recs added
